src/helpers.c

```diff
@@ -33,6 +34,14 @@ char *combine_path_name(const char *file_path, const char *file_name) {
   return combined;
 }
 
+// Set default value defined in include/defines.h
+void set_default_value(Key_File *key_file, size_t num) {
+  const size_t NULL_LENGTH = strlen(NULL_VALUE) + 1;
+  snprintf(key_file->file_entry[num].group = malloc(NULL_LENGTH), NULL_LENGTH, NULL_VALUE);
+  snprintf(key_file->file_entry[num].key = malloc(NULL_LENGTH), NULL_LENGTH, NULL_VALUE);
+  snprintf(key_file->file_entry[num].value = malloc(NULL_LENGTH), NULL_LENGTH, NULL_VALUE);
```

Comment:
Why not using strdup instead of snprintf?

Reply:
Another comment about the function name: It will only initialize Key_File with NULL values - so why is it called `set_default_value` and not something with `initialize`?